### ISSUE TITLE
Ansible with the bastion

### ISSUE THREAD

**Issue by mvalois** [ovh/the-bastion#105]
Firstly, thank you for having opened the code of the bastion! This is such a great tool :)

Provided the fact that the bastion is not a proxy jump, how could we use solutions like Ansible to run playbooks over our infrastructure?

**Comment by speed47** [comment]
Hello!

> Firstly, thank you for having opened the code of the bastion! This is such a great tool :)

Nice to hear it can be of use to others!

> Provided the fact that the bastion is not a proxy jump, how could we use solutions like Ansible to run playbooks over our infrastructure?

This is possible by using wrappers. We do have those internally, they were previously in this very repo in the "contrib" directory (before we pushed it to GitHub), but as other teams needed to use this wrapper without checking out all The Bastion source, we moved it to its own repo. We'll be open-sourcing those then! Maybe next week or early January if we can't get enough people for the open-sourcing vetting process during the holidays!

In any case, stay tuned. :)
